Cargo.toml

```diff
@@ -19,7 +19,5 @@ num = "0.4.0"
 rand = "0.8.1"
 serde = { version = "1.0.118", features = ["derive"] }
 sha2 = "0.9.2"
-zbus = "1.7.0"
-zbus_macros = "1.7.0"
-zvariant = "2.4.0"
-zvariant_derive = "2.4.0"
+zbus = "2"
+zvariant = "3"
```

Comment:
Looking at the rest of the dependencies, this one should probably be "3.0.0", or "3.0" at worst. Ditto for `zbus`

Reply:
This is correct to my knowledge as well. Specifying only a major version with no range exceptions will allow `cargo` to update as desired and pick the latest version with no dependency conflicts.